### PR TITLE
Update config-settings.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Updates for: https://github.com/mattermost/docs/issues/3845

### CODE REVIEW

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -5210,6 +5212,12 @@ When running Mattermost on a single machine, this setting should always be enabl
 
 When running Mattermost in High Availablity mode, this setting should always be enabled. In a High Availability cluster, exactly one of the servers will be designated as the Scheduler at a time to ensure that duplicate tasks aren't created. See `High Availability documentation <https://docs.mattermost.com/deployment/cluster.html#job-server>`__ for more details.
 
+.. warning::
```
Comment (amyblais) [inline source/administration/config-settings.rst:5215]: The warning label is not showing up. Perhaps it's an alignment issue

**Comment by justinegeffen** [comment]
Yep, thanks, @sadohert, that was the main review area. :)

File: source/administration/config-settings.rst (hunk 1 above)
Comment (justinegeffen) [reply source/administration/config-settings.rst:5215]: Should be fixed now, thanks!

**Comment by amyblais** [comment]
Looks good although I noticed that the TOC has mysteriously disappeared from ``administration/config-settings.rst``. I don't see any direct changes in this PR that would have led to it.

Screenshot at Aug 19 12-23-06

**Comment by justinegeffen** [comment]
Thanks, @amyblais. I made the "Configuration in database" and "Environment variables" into sections - they were above the TOC entry though, so it looks like that caused the TOC to disappear. Sorted now.

I'd like to move the environment variables section to a more prominent position - that's a separate PR though.

**Pull request merged**